src/spans/awsSpan.ts

```diff
@@ -238,10 +238,10 @@ export const decodeHttpBody = (httpBody: any, hasError: boolean): any | string =
   return httpBody;
 };
 
-export const getHttpInfo = (requestData, responseData, truncated = false): HttpInfo => {
+export const getHttpInfo = (requestData, responseData): HttpInfo => {
   const { host } = requestData;
   const request = Object.assign({}, requestData);
-  const response = Object.assign({ truncated }, responseData);
+  const response = Object.assign({}, responseData);
```

Comment:
I'm not sure what is the purpose of this line - is this a copy?

Reply:
we moved the truncated field to another place.